interface.go

```diff
@@ -84,6 +84,11 @@ type Job interface {
 	// exclusion a job can provide.
 	Scopes() []string
 	SetScopes([]string)
+
+	// ApplyScopesOnEnqueue allows the scope exclusion functionality to be
+	// configured so that exclusion occurs during job dispatch or enqueue.
+	ApplyScopesOnEnqueue() bool
```

Comment:
This is a bit confusing cause `apply` here can also sound like a verb. I almost want to change this to `IsApplyScopesOnEnqueueSet` or `IsApplyScopesOnEqueue` but maybe that is too long. What do you think?

Reply:
I agree, `Should` makes the most sense.

Reply:
Done.